pkg/controller/util.go

```diff
@@ -91,6 +91,23 @@ func getSecretName(client kubernetes.Interface, pvc *v1.PersistentVolumeClaim) (
 	return name, nil
 }
 
+// returns the import content.
+func getImportContent(pvc *v1.PersistentVolumeClaim) (string, error) {
```

Comment:
Sorry, I'm not completely following what's going on in the logic below here?  Also, it seems like we could just declare a variable to represent this map and set it (it's just joining the existing constants we set).  Maybe I'm missing something, but this seems like a lot of churn just to get the string back from the pvc content annotation.  I'm sorry if I'm missing this.

Reply:
Irrelevant anymore after rebasing on top of #520.